inst/extdata/database/tedfs/Tech/Coal Gasification.csv

```diff
@@ -1,17 +1,17 @@
-92.50%,Input|Coal,Output|Hydrogen,2023,467308.0,,lb,60627.0,lb,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 5"
-0.00%,Input|Electricity,Output|Hydrogen,2023,25.0,,MWh,60622.0,lb,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 4"
-92.50%,Input|Electricity,Output|Hydrogen,2023,39.0,,MWh,60627.0,lb,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 5"
-0.00%,Capture Rate,,2023,0.0,,percent,,,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 4"
-92.50%,Capture Rate,,2023,92.5,,percent,,,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 5"
+subtech,variable,reference_variable,period,value,uncertainty,unit,reference_value,reference_unit,comment,source,source_detail
+Standard,CAPEX,Output Capacity|Hydrogen,2023,5853.0,1170.6,USD_2018,1.0,kg/day,Using Total As-Spent Cost; Source reports -15%/+25% uncertainty in Exhibit 2-22. We use +-20% here.,Lewis22,"Section 5.2, Exhibit 5-7, Printed page 273 (document page 291/327), Case 4"
+w/ CC (92.5%),CAPEX,Output Capacity|Hydrogen,2023,6747.0,1349.4,USD_2018,1.0,kg/day,Using Total As-Spent Cost; Source reports -15%/+25% uncertainty in Exhibit 2-22. We use +-20% here.,Lewis22,"Section 5.2, Exhibit 5-7, Printed page 273 (document page 291/327), Case 5"
+Standard,OPEX Fixed,Output Capacity|Hydrogen,2023,101984658.0,,USD_2018/year,659947.0,kg/day,,Lewis22,"Section 4.8, Exhibit 4-31, Printed page 193 (document page 211/327)"
+w/ CC (92.5%),OPEX Fixed,Output Capacity|Hydrogen,2023,116475905.0,,USD_2018/year,660004.0,kg/day,,Lewis22,"Section 4.12, Exhibit 3-33, Printed page 226 (document page 244/327)"
+Standard,OPEX Variable,Output|Hydrogen,2023,0.43269,,USD_2018,1.0,kg,,Lewis22,"Section 4.8, Exhibit 4-31, Printed page 193 (document page 211/327)"
+w/ CC (92.5%),OPEX Variable,Output|Hydrogen,2023,0.03493,,USD_2018,1.0,kg,,Lewis22,"Section 4.12, Exhibit 3-33, Printed page 226 (document page 244/327)"
+Standard,Input|Coal,Output|Hydrogen,2023,467308.0,,lb,60622.0,lb,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 4"
+w/ CC (92.5%),Input|Coal,Output|Hydrogen,2023,467308.0,,lb,60627.0,lb,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 5"
+Standard,Input|Electricity,Output|Hydrogen,2023,25.0,,MWh,60622.0,lb,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 4"
+w/ CC (92.5%),Input|Electricity,Output|Hydrogen,2023,39.0,,MWh,60627.0,lb,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 5"
+Standard,Emission|CO2,Output|Hydrogen,2023,199,,lb,1,MMBtu,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 4"
+w/ CC (92.5%),Emission|CO2,Output|Hydrogen,2023,15,,lb,1,MMBtu,,Lewis22,"Results Analysis, Exhibit ES-2, Printed page 6 (document page 24/237), Case 5"
```

Comment:
I was trying to be as consistent as possible across files so that risks of double-counting are reduced down the line. What I don't like about capture rate is that it masks the total (higher) CO2 emissions that CC options have, and where emissions come from if they are not just energy-based. 
My preferred way is to do it like [in the cement file](https://github.com/clarabachorz/posted/blob/9b84e34fc4a1e891fc88abcd929a56cbe36493b8/inst/extdata/database/tedfs/Tech/Cement%20Production.csv#L38-L49), where CO2 emissions and CO2 captured are reported. But in the coal gasification case, I can keep both variables instead.

Reply:
Thanks for explaining and for pointing me to this.

* First, regarding the emission variables you reported in the cement file, shouldn't it be `Emissions|CO2` and `Output|Captured CO2` (rather than `Emissions|CO2 Captured`)?
* I get that it's a little trickier with cement, because we don't report the source of carbon so clearly (i.e. we would have to report the carbon content of minerals used to be able to calculate specific emissions just fro the capture rate).
* I agree that the specific emissions are really good to have, yet the capture rate may also come in handy when we do sensitivity analysis.
* Here is my conclusion: let's report specific emissions and captured CO2 only, when they are available, and otherwise capture rate. We could use variable mappings to get the missing variables automatically from POSTED.

Does that make sense?

Reply:
Sure, changed the files accordingly